templates/kubernetes/features.html

```diff
@@ -68,44 +68,40 @@ <h3 class="p-card__title">Managed Kubernetes</h3>
   </div>
 </section>
 
-<section class="p-strip is-bordered" id="bare-metal">
+
+<section class="p-strip">
   <div class="row">
-    <div class="col-8">
-      <h2>Charmed Kubernetes fits perfectly on top of OpenStack, VMware, and bare metal</h2>
-      <p>There's no doubt that Kubernetes is the new standard operational layer for every multi-cloud business. Canonical enables K8s on demand for your DevOps teams &mdash; on OpenStack, VMware, public clouds, and bare metal clusters with MAAS.</p>
+    <div class="col-6">
+      <h2>Shape it to your needs</h2>
     </div>
   </div>
   <div class="row">
-    <div class="col-8">
-      <p>With Charmed Kubernetes, you can deliver 'Containers as a Service' across the enterprise, to enable each project to spin up a standardised K8s environment of arbitrary scale, on demand, with centralised operational control.</p>
+    <div class="row u-equal-heigh">
```

Comment:
There shouldn't be a row nested in a row (you only need one). Also - spelling mistake:
```suggestion
    <div class="row u-equal-height">
```

Reply:
Oh my bad sorry for the typo, I will look into the nested row, thanks :)